### ISSUE TITLE
Option to exclude particular packages

### ISSUE THREAD

**Issue by lubomir** [fedora-modularity/fus#3]
We need to be able to tell the code to exclude particular packages. It should support globs against package names.

The behaviour is that if a package name matches the glob, it should be completely invisible, as if it was not in the repo at all.

**Comment by contyk** [comment]
And this needs to be limited to ursine RPMs. Modules have their own filters.

**Comment by igor-raits** [comment]
What about lookaside?

> --

-Igor Gnatenko

**Comment by lubomir** [comment]
Lookaside should not be modified. This is meant to avoid getting a particular package into the compose, and packages from lookaside will never be pulled in (they are in the printed results but will be ignored anyway).

**Comment by igor-raits** [comment]
If it meant to **not*" get package in compose, then we should exclude
package from lookaside too.. and from modules.

> --

-Igor Gnatenko

**Comment by lubomir** [comment]
I don't think so. For modules the filters should be applied at the modulemd level, which is separate. Lookaside is available to consumers of the compose as another repo, and they will not know what are the filters used in composes. Applying the filter to lookaside could only result in either false alarm about broken deps, or in pulling in different package that is not actually needed.

I checked current configs, and the hybrid parts don't actually use this yet, so I removed the blocker label.
